src/server/db/user.ts

```diff
@@ -4,6 +4,7 @@ import { Doc } from 'src/types/common'
 export interface User extends Doc {
   name: string
   studentID?: string
+  allowDocumentation?: boolean
```

Comment:
one last thing! let's make the name consistent with the name in api `dataCollectionConsentState`

Reply:
> one last thing! let's make the name consistent with the name in api `dataCollectionConsentState`

Ahh yes updated here https://github.com/kixlab/kuiz/pull/59/commits/32f0c91963cb3881b00da4d89e6b966d90d7b3f2 :)